ci/test/resources/build.yaml

```diff
@@ -7,7 +7,7 @@ steps:
    name: inline_image
    dockerFile:
      inline: |
-       FROM gcr.io/{{ global.project }}/ubuntu:18.04
+       FROM {{ global.region }}-docker.pkg.dev/{{ global.project }}/hail/ubuntu:18.04
```

Comment:
how come this differs from everything else with `docker_prefix`?

Reply:
My bad!